ADDITIONS.txt

```diff
@@ -0,0 +1,4 @@
+* Manually change CP
+* Upgrade ships
+* Allow for "no maintenance" (Ameeoba scenario, etc)
+* Allow for random space wrech technology
```

Comment:
What do you think if we create a CHANGE.log instead?
https://keepachangelog.com/en/1.0.0/

Something like this.
https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md

Reply:
A CHANGELOG.md file would be good; I can add that.  This file was actually meant for me to track things I wanted to add later :)  More of a "todo" list.  We can remove it from the repo.  I'll do that in a PR update soon.